e2e/testgames/game/Dockerfile

```
@@ -17,12 +17,36 @@
+FROM golang:1.22-bookworm AS runtime-debug
+
+WORKDIR /go/src/app
+
+COPY e2e/testgames e2e/testgames
+COPY cardinal cardinal
+COPY assert assert
+
+# Install delve
+RUN go install github.com/go-delve/delve/cmd/dlv@latest
+
+RUN cd e2e/testgames && go mod download
```

Hadolint

[warning] Use WORKDIR to switch to a directory

(DL3003)